### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by Muffexx** [Flow-Launcher/Flow.Launcher#497] [General]
Hello all,

I have been using Wox for years. Since the development is discontinued, I switched to Flow-Launcher.
I love flow launcher in combination with the "Everything" plugin as it is truly useful.
Is there any way to donate something to you guys?

I hope the tool will remain free for all.

Regards
Muffexx

**Comment by jjw24** [comment]
Hi @Muffexx, thank you for considering donating to Flow Launcher.

Flow will always be free and shaped by the community.

My own perspective on donation is that it will obviously go towards helping with Flow's own expenses such as hosting and etc. At this stage if we get more traction in the donation space we could definitely set something up.

Obviously just my thoughts so far, definitely open to suggestions and recommendations too.

**Comment by jjw24** [comment]
Just a note here that if you would like to contribute financially, flow has joined the [GitHub sponsorship](https://github.com/sponsors/Flow-Launcher) program and is also on [Open Collective](https://opencollective.com/flow-launcher).
